EIPS/eip-5252.md

```diff
@@ -0,0 +1,53 @@
+requires: 721, 1155, 5114
+created: 2022-06-29
+---
+
+## Abstract
+This EIP proposes a form of smart contract design pattern and a new type of account abstraction on how one's finance should be managed in web3.0, ensuring transparency of managing investments and protection with self-sovereignty even from its financial operators.
+
+This EIP also extends account-bound token from [EIP-5114](https://eips.ethereum.org/EIPS/eip-5114), but the difference is that it can be transferred to other accounts for mobility between managing multiple wallets.
+
+## Motivation
+
+Finance in the crypto ecosystem is facing a huge trust issue. Smart contracts can now be a proxy of multisig for operating teams, and Insolvencies exist in that there is no trace of funds written in the blockchain. That can create an arbitrary loss of funds, as happened in the official bankruptcy announcement of Celsius and UST depegging and anchor protocol failure. Smart contract should be used for peer-to-peer only, with immutable rules etched so that only one who has the private key has full access to managing his investment as bitcoin promises. 
+
+The pattern empowers more self-sovereignty and gives more credentials(obviously with an Account-bound token) than locking financial data in the operating team's contract.
+
+Segregation between investor's fund and operation fee is clearly specified in the smart contract, so investors can ensure safety from arbitrary loss of funds by the operating team's control.
+
+## Specification
+
+The pattern consists of 5 components, manager, factory, finance, account-bound token, and extension.
+
+### Diagram
+
+TBD
+
+**`Manager`**: **`Manager`** contract acts as an entry point to interact with the investor. The contract also stores parameters for **`Finance`** contract.
+
+**`Factory`**: **`Factory`** contract manages contract bytecode to create for managing investor's fund and [clones](https://soliditydeveloper.com/clonefactory/) **`Finance`** contract on **`Manager`** contract's approval. It also mints account-bound tokens to interact with the `Finance` contract.
+
+**`Finance`**: **`Finance`** contract specifies all rules on managing an investor's fund. The contract is only accessible with an account that has an Account-bound token. When an investor deposits a fund to **`Manager`** contract, the contract sends the fund to **`Finance`** contract accont after separating fees for operation. 
+
+**`Account-bound token`**: **`Account-bound token`** contract in this EIP can bring the **`Finance`** contract's data and add metadata. For example, if there is a money market lending **`Finance`** contract, its **`Account-bound token`** can show how much balance is in agreement using SVG.
+
+**`Extension`**: **`Extension`** contract is another contract that can utilize locked funds in **`Finance`** contract. The contract can access with **`Finance`** contract on operator's approval managed in **`Manager`** contract.
+
+## Reference Implementation
```

Comment:
This must come after the backward compatibility section.

Reply:
If I do this, then ci error happens. So I am lost here.